rest-server/src/config/db.js

```diff
@@ -0,0 +1,28 @@
+import mysql from 'mysql';
+import { success, error } from '../lib/log'
```

Comment:
Same as before. Separate lines unless we know 100% it's only one import and will only ever be one